IOPool/Streamer/interface/FRDEventMessage.h

```diff
@@ -57,14 +69,22 @@
 
 #include "IOPool/Streamer/interface/MsgTools.h"
 
+struct FRDEventHeader_V6 {
+  uint16 version_;
+  uint16 flags_;
+  uint32 run_;
+  uint32 lumi_;
+  uint32 event_;
+  uint32 eventSize_;
+  uint32 crc32c_;
+};
+
 struct FRDEventHeader_V5 {
   uint32 version_;
   uint32 run_;
   uint32 lumi_;
-  uint32 eventLow_;
```

Comment:
is it OK to change the "v5" specification here ?

Reply:
Yes, I cross-checked with the EvB code and apparently v5 was not defined correctly here. event ID here is 32-bit, and padding value was removed. That struct is (fortunately) unused, so it is for reference only.